src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestHelpers/TestRunner.cs

```diff
@@ -377,6 +375,12 @@ internal static void TestToString(FormulaValue result, StringBuilder sb)
             {
                 sb.Append("Blank()");
             }
+            else if (result is DateValue d)
```

Comment:
Are we going to add Time or DateTime as part of a follow-up?  Or should we build out that infra now?